src/plugins/Polls/Services.ts

```diff
@@ -0,0 +1,151 @@
+import Gun from 'gun'
+import 'gun/lib/then.js'
+import type { PollMetaData } from './types'
+import { PluginMessageCenter } from '../PluginMessages'
+import { gun2 } from '../../network/gun/version.2'
+import { first } from 'lodash-es'
+
+const gun = gun2
+const PollGun = gun.get('com.maskbook.plugin.polls')
+
+const defaultPoll: PollGunDB = {
+    key: '',
+    sender: '',
+    question: '',
+    start_time: Date.now(),
+    end_time: Date.now(),
+    options: ['', ''],
+    results: [0, 0],
+}
+
+interface NewPollProps {
+    sender?: string | undefined
+    id?: string | undefined
+    question: string
+    options: Array<string>
+    start_time: Date
+    end_time: Date
+}
+
+export async function createNewPoll(poll: NewPollProps) {
+    const { id, options, start_time, end_time } = poll
+
+    const results = new Array<number>(options.length).fill(0)
+    const resultsObj: Object = { ...results }
+    const optionsObj: Object = { ...options }
+
+    const poll_item = {
+        ...poll,
+        results: resultsObj,
+        options: optionsObj,
+        start_time: start_time.getTime(),
```

Comment:
plz use `Date.prototype.toISOString()` (ISO 8601 formatted)

Reply:
this data is posted to the web IMO, does it any reason to not use UNIX timestamp?

Reply:
why use ISO 8601 formatted? I'm intended to store UNIX timestamp data into GunDB